README.md

```diff
@@ -31,7 +31,7 @@ Smee.io is a simple Node.js application. You can deploy it any way you would dep
 Don't forget to point `smee-client` to your instance of `smee.io`:
 
 ```
-smee --url https://your-smee.io/channel 
+smee --url https://smee.io/<your-channel>
```

Comment:
It says

> to _your_ instance of `smee.io`

and smee.io is our instance, not yours :)

Reply:
@gr2m I disagree.. the URL is smee.io but the README can be wrongly interpreted as your-smee.io/channel which doesn't sound right to me :)

Reply:
The whole section is titled [Deploying your own Smee.io](https://github.com/probot/smee.io#deploying-your-own-smeeio). It's about deploying your own instance. Maybe we can make the wording more clear if it isn't to you, but that would be an update to the README. The code you changed is the way we intend it to be